README.md

```diff
@@ -4,51 +4,30 @@
 
 Utilities to redact sensitive data in Elixir projects
 
-## Run locally
+## Redacting
 
-### Setup Auth0
+By *redacting* we consider here the practice of hiding part of a potentially sensitive information, while keeping a small part
+of it to allow understanding that it may be a valid one.
 
-To access some endpoints, queries and/or mutations, requests will need to be authenticated via a JWT.
+For example, you may want to *redact* a telephone number in your logs, but keep the initial digits (e.g. the prefix) because that information is relevant
+to  your understanding of the system, and at the same moment it does not expose any sensitive information.
 
-In staging and production-like environment this will be handled via Auth0.
-However, to simplify local development, on the local dev environment JWTs are accepted even when their signature is not valid (only a warning will be logged on invalid JWT signature).
-This means that it is possible to use locally forged JWT tokens for local instances of this application.
+## Configuration
 
-To forge a new token for local development open a `iex` shell with `iex -S mix` and run
+## Usage
 
-```elixir
-PrimaAuth0Ex.LocalToken.forge("redact_ex", %{permissions: ["some:permission"]})
-```
+See [RedactEx.Redacter](./lib/redact_ex/redacter.ex) for generating fast redacters for strings
+See [RedactEx.Redactable](./lib/redact_ex/redactable.ex) protocol for deriving redactable structs
 
-### Run
+## TODO
 
-The recommended way to work locally is to spawn a shell inside a Docker container with the following command:
+   [ ] Switch to enable/disable masking per environment
+   [ ] Expand explanation of fallback custom implementations of algorithms
+   [ ] fix dialyzer :(
```

Comment:
```suggestion
- [ ] Switch to enable/disable masking per environment
- [ ] Expand explanation of fallback custom implementations of algorithms
- [ ] fix dialyzer :(
```

This will fix the checkboxes :D